data/templates/nginx/plain/yunohost_admin.conf

```diff
@@ -19,6 +19,7 @@ server {
     ssl_certificate_key /etc/yunohost/certs/yunohost.org/key.pem;
     ssl_session_timeout 5m;
     ssl_session_cache shared:SSL:50m;
+    ssl_ecdh_curve secp521r1:secp384r1:prime256v1;
```

Comment:
Uh could you elaborate on where those come from ? For instance for the ciphers we cite the mozilla source which is kinda trustable, since all those things are touchy security stuff...

Also how do you know this related to the DH params ? Does that mean we don't need the ssl_dhparam thingy at all ? Or are those DH params but only for eliptic curve ciphers ?

Reply:
Just read this : https://wiki.mozilla.org/Security/Server_Side_TLS ;-)

Reply:
It was just to change the description from just "security problem"...

Reply:
It's not cyphers btw....

Reply:
Alrighty, I just moved the instructions right next to the other cipher-related things, since that's the same reference ;)

Reply:
euh... but these are the same ecdh crurves for the two... modern or intermediate... so I thought/think it's better to set it at the top. like this the final user will be completely lost and perhaps not enable this.